uxexperience/settings.py

```diff
@@ -106,8 +105,11 @@
 ]
 
 # Heroku DB configuration
-db_from_env = dj_database_url.config(conn_max_age=500)
-DATABASES['default'].update(db_from_env)
+try: import dj_database_url
```

Comment:
but if you run pip install -r requirements.txt in your virtualenv you should install those packages so it shouldn't be an issue.